plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ScopeIndex.kt

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2010-2025 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.formver.conversion
+
+sealed class ScopeIndex {
```

Comment:
Probably past time to do this!  But please add a comment as to the purpose of the class, so it doesn't need to be reverse engineered from usage.